src/components/MemberSearchPage/MemberSearchBioStory.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import FormatLabel from 'components/MemberSearchPage/FormatLabel';
+import { Divider, Typography } from 'antd';
+import { bind } from '../../utils';
+import PropTypes from 'prop-types';
+
```

Comment:
The CircleCI build fails ATM
https://circleci.com/gh/kids-first/kf-portal-ui/2292?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

```
Failed to compile.

./src/components/MemberSearchPage/MemberSearchBioStory.js
  Line 1:  'Component' is defined but never used  no-unused-vars
  Line 4:  'bind' is defined but never used       no-unused-vars
  Line 5:  'PropTypes' is defined but never used  no-unused-vars
```